### PR TITLE
Add `ignore_timestamps_errors` to convert_session for OpenEphys

### LINKED ISSUE #29: Discontinuous timestamps in AUX.continuous data files
Working subject 41, I received this error:
(tye_lab_to_nwb_env) lkeyes@node32:~/Projects/GIT/tye-lab-to-nwb$ python src/tye_lab_to_nwb/neurotensin_valence/neurotensin_valence_convert_all_sessions.py
  0%|                                                                                                                                                | 0/1 [00:00<?, ?it/s]                                  Source data is valid!
concurrent.futures.process._RemoteTraceback:
"""
Traceback (most recent call last):
  File "/home/lkeyes/anaconda3/envs/tye_lab_to_nwb_env/lib/python3.11/concurrent/futures/process.py", line 256, in _process_worker
    r = call_item.fn(*call_item.args, **call_item.kwargs)
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nadata/snlkt/home/lkeyes/Projects/GIT/tye-lab-to-nwb/src/tye_lab_to_nwb/neurotensin_valence/neurotensin_valence_convert_session.py", line 145, in session_to_nwb
    converter = NeurotensinValenceNWBConverter(source_data=source_data)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nadata/snlkt/home/lkeyes/Projects/GIT/tye-lab-to-nwb/src/neuroconv/src/neuroconv/nwbconverter.py", line 64, in __init__
    self.data_interface_objects = {
                                  ^
  File "/nadata/snlkt/home/lkeyes/Projects/GIT/tye-lab-to-nwb/src/neuroconv/src/neuroconv/nwbconverter.py", line 65, in <dictcomp>
    name: data_interface(**source_data[name])
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nadata/snlkt/home/lkeyes/Projects/GIT/tye-lab-to-nwb/src/neuroconv/src/neuroconv/datainterfaces/ecephys/openephys/openephysdatainterface.py", line 40, in __new__
    return OpenEphysLegacyRecordingInterface(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nadata/snlkt/home/lkeyes/Projects/GIT/tye-lab-to-nwb/src/neuroconv/src/neuroconv/datainterfaces/ecephys/openephys/openephyslegacydatainterface.py", line 42, in __                                  init__
    available_streams = self.get_stream_names(folder_path=folder_path)
                        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/nadata/snlkt/home/lkeyes/Projects/GIT/tye-lab-to-nwb/src/neuroconv/src/neuroconv/datainterfaces/ecephys/openephys/openephyslegacydatainterface.py", line 57, in ge                                  t_stream_names
    stream_names, _ = OpenEphysLegacyRecordingExtractor.get_streams(folder_path=folder_path)
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/lkeyes/anaconda3/envs/tye_lab_to_nwb_env/lib/python3.11/site-packages/spikeinterface/extractors/neoextractors/neobaseextractor.py", line 142, in get_streams
    neo_reader = get_reader(cls.NeoRawIOClass, **neo_kwargs)
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/lkeyes/anaconda3/envs/tye_lab_to_nwb_env/lib/python3.11/site-packages/spikeinterface/extractors/neoextractors/neobaseextractor.py", line 14, in get_reader
    neo_reader.parse_header()
  File "/home/lkeyes/anaconda3/envs/tye_lab_to_nwb_env/lib/python3.11/site-packages/neo/rawio/baserawio.py", line 179, in parse_header
    self._parse_header()
  File "/home/lkeyes/anaconda3/envs/tye_lab_to_nwb_env/lib/python3.11/site-packages/neo/rawio/openephysrawio.py", line 118, in _parse_header
    assert np.all(diff == RECORD_SIZE), \
AssertionError: Not continuous timestamps for 104_AUX1.continuous. Maybe because recording was paused/stopped.
"""

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/nadata/snlkt/home/lkeyes/Projects/GIT/tye-lab-to-nwb/src/tye_lab_to_nwb/neurotensin_valence/neurotensin_valence_convert_all_sessions.py", line 92, in <module>
    parallel_convert_sessions(
  File "/nadata/snlkt/home/lkeyes/Projects/GIT/tye-lab-to-nwb/src/tye_lab_to_nwb/neurotensin_valence/neurotensin_valence_convert_all_sessions.py", line 83, in parallel_con                                  vert_sessions
    future.result()
  File "/home/lkeyes/anaconda3/envs/tye_lab_to_nwb_env/lib/python3.11/concurrent/futures/_base.py", line 449, in result
    return self.__get_result()
           ^^^^^^^^^^^^^^^^^^^
  File "/home/lkeyes/anaconda3/envs/tye_lab_to_nwb_env/lib/python3.11/concurrent/futures/_base.py", line 401, in __get_result
    raise self._exception
AssertionError: Not continuous timestamps for 104_AUX1.continuous. Maybe because recording was paused/stopped.

### PR DESCRIPTION
**Pull request by weiglszonja**
This should resolve #29 (and #28) when neuroconv is installed from 
```bash
 pip install git+https://github.com/catalystneuro/neuroconv@36fe8ba02c036e715a915ee66079391a50536fb2
```

### CODE REVIEW

File: src/tye_lab_to_nwb/ast_ecephys/convert_session.py [hunk 1]
```diff
@@ -183,7 +190,7 @@ def session_to_nwb(
 
     # For faster conversion, stub_test=True would only write a subset of ecephys and plexon data.
     # When running a full conversion, use stub_test=False.
-    stub_test = False
+    stub_test = True
```
Comment (weiglszonja) [inline src/tye_lab_to_nwb/ast_ecephys/convert_session.py:193]:
```suggestion
    stub_test = False
```

**Comment by CodyCBakerPhD** [comment]
@weiglszonja Should we go ahead and pin to that commit hash on the repo (or project) requirements?

**Comment by weiglszonja** [comment]
@CodyCBakerPhD I agree, I pinned to `add_ignore_timestamps_errors` branch.

**Comment by weiglszonja** [comment]
@CodyCBakerPhD good to merge?

**Review by CodyCBakerPhD — APPROVED** [review]

**Pull request merged**